.github/workflows/ESlint.yaml

```
@@ -0,0 +1,47 @@
+name: ESlint
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+
+jobs:
+  code:
+    name: Versões NodeJS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 14.x
+          - 15.x
```

O outro workflow utiliza a versão 12 do node, o que acha de manter o padrão?
```suggestion
          - 12.x
```